### PR TITLE
[WIP] Add default nginx container for services

### PR DESCRIPTION
**Pull request by kenoir**
In order to make the process of setting up a service faster this PR adds a default nginx container hosted on docker hub that does not require http domain and application port to be hard coded.

### CODE REVIEW

File: services_default_nginx/nginx.conf.template [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+    # Ensure that requests sent over HTTP are redirected to HTTPS.
+    # http://scottwb.com/blog/2013/10/28/always-on-https-with-nginx-behind-an-elb/
+    proxy_set_header X-Real-IP $remote_addr;
+    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+    proxy_set_header Host $http_host;
+    proxy_redirect off;
+    proxy_next_upstream error;
+
+    if ($http_x_forwarded_proto != "https") {
```
Comment (jamesgorrie) [inline services_default_nginx/nginx.conf.template]: https://www.nginx.com/resources/wiki/start/topics/depth/ifisevil/

Comment (kenoir) [reply services_default_nginx/nginx.conf.template]: > Directive if has problems when used in location context

So we're good?

File: services_default_nginx/nginx.conf.template [hunk 2]
```diff
@@ -0,0 +1,37 @@
+worker_processes 1;
+
+events { worker_connections 1024; }
+
+http {
+  server {
+    listen 9000;
```
Comment (jamesgorrie) [inline services_default_nginx/nginx.conf.template]: How're are we listening on port 9000?

**Comment by kenoir** [comment]
Closing as this really belongs in the dockerfiles repo.

**Pull request closed without merging**